cit-api/pipeline/management/commands/send-user-tracking.py

```diff
@@ -0,0 +1,85 @@
+from django.core.management.base import BaseCommand
+from django.db import connection
+from datetime import date, timedelta
+import csv
+import os
+import requests
+import json
+from io import StringIO
+from requests.auth import HTTPBasicAuth
+import calendar
+import base64
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        send_tracking_email()
+
+def send_tracking_email():
+    token_request_body = {'grant_type': 'client_credentials'}
+    response = requests.post(os.environ.get("EMAIL_AUTH_HOST") + "/auth/realms/jbd6rnxw/protocol/openid-connect/token",
+                             data=token_request_body, auth=HTTPBasicAuth(os.environ.get("EMAIL_CLIENT_ID"), os.environ.get("EMAIL_CLIENT_SECRET")))
+    if response.status_code == 200:
```

Comment:
We should have an else here in case there is no 200 response. If we print a message, we'll be able to see what the problem was in the logs.

Reply:
Updated in the else below